CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v1.5.0]
+
+### BREAKING CHANGES
+
+- **CUMULUS-1629**
+  - Update image python to python3.   This version of cumulus-ecs-task is *required* for CMA >= 1.2.0.
```

Comment:
Nit: can we link the text "CMA" to the repo for clarity?

Reply:
Certainly 👍

Reply:
da4df30